easybuild/easyconfigs/g/GL2PS/GL2PS-1.4.0-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'CMakeMake'
+
+name = 'GL2PS'
+version = '1.4.0'
+
+homepage = 'https://www.geuz.org/gl2ps/'
+description = """GL2PS: an OpenGL to PostScript printing library"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+source_urls = ['https://geuz.org/gl2ps/src/']
+sources = [SOURCELOWER_TGZ]
+checksums = ['03cb5e6dfcd87183f3b9ba3b22f04cd155096af81e52988cc37d8d8efe6cf1e2']
+
+builddependencies = [
+    ('CMake', '3.15.3'),
+    ('binutils', '2.32'),
+]
+
+dependencies = [
+    ('X11', '20190717'),
+    ('Mesa', '19.1.7'),
+    ('libGLU', '9.0.1'),
+    ('freeglut', '3.2.1'),
+    ('libpng', '1.6.37'),
+    ('zlib', '1.2.11'),
+]
+
+sanity_check_paths = {
+    'files': ['include/gl2ps.h', 'lib/libgl2ps.so'],
```

Comment:
`'lib/libgl2ps.%s' % SHLIB_EXT` would be more portable

Reply:
Changed in last commit